tests/e2e/scripts/gke/deploy.sh

```diff
@@ -121,6 +122,40 @@ fi
 run kubectl create -f ${YAML_FILE} --namespace "${NAMESPACE}"
 HOST=$(get_cluster_host "${NAMESPACE}")
 
+# Run in background while e2e tests are running.
+# ESPv2 is deployed in managed mode for all these e2e tests.
+# This will cause ESPv2 to rebuild the Envoy listener while lots of traffic is running through.
+function doServiceRollout() {
+  if [ "${BACKEND}" != 'bookstore' ]; then
+    echo "doServiceRollout: TODO(nareddyt): Support managed service rollout for ${BACKEND}"
+    return 0
+  fi
+
+  while true; do
+    echo 'doServiceRollout: Sleeping until next service rollout'
+    sleep 15m
```

Comment:
If this test only cares about crash, is the period too long? 
I checked the running part of all tests are less than 30mins so the config will be updated once.

Reply:
For our presubmits, yet. But we mostly care about this during our long run tests. Then it will be updated many times.

If we update it more often than 15 minutes, we may have conflicts between different test runs. They all use the same service.

Reply:
I see.